src/main/java/com/commerce/backendserver/member/domain/Member.java

```diff
@@ -7,27 +7,45 @@
 import static jakarta.persistence.GenerationType.IDENTITY;
 import static lombok.AccessLevel.PROTECTED;
 
-@Builder
 @Entity
 @Getter
-@Setter
-@AllArgsConstructor
 @NoArgsConstructor(access = PROTECTED)
-@Table(name = "t_user")
+@Table(name = "t_member")
 public class Member extends BaseEntity {
 
     @Id
     @GeneratedValue(strategy = IDENTITY)
     @Column(name = "member_id")
     private Long id;
 
-    private String nickName;
+    @Column(unique = true)
+    private String oauthId;
 
-    public static Member of(
-            String nickName
+    private String oauthType;
+
+    private String nickname;
+
+    //생성 메소드
+    private Member(
+            String nickName,
+            String oauthId,
+            String oauthType
     ) {
-        return Member.builder()
-                .nickName(nickName)
-                .build();
+        this.nickname = nickName;
+        this.oauthId = oauthId;
+        this.oauthType = oauthType;
+    }
+
+    public static Member createMember(
+            String nickName,
+            String oauthId,
+            String oauthType
+    ) {
+        return new Member(nickName, oauthId, oauthType);
+    }
```

Comment:
저 같은 경우에는 이런 상황에서 빌더 패턴을 즐겨 사용하는 편이에요. 파라미터에 대응하는 변수를 직관적으로 볼 수 있기 때문이라 즐겨 사용합니다. 빌더 패턴에 대해서는 어떻게 생각하시는지 궁금합니다.

```suggestion
    @Builder
    public static Member createMember(
            String nickName,
            String oauthId,
            String oauthType
    ) {
        return Member.builder()
                .nickName(nickName)
                .oauthId(oauthId)
                .oauthType(oauthType)
                .build();
    }
```

Reply:
괜찮은거같아요 적용해볼게요